conda-forge.yml

```
@@ -13,3 +13,5 @@ provider:
   linux_ppc64le: default
   osx_arm64: default
 test: native_and_emulated
+azure:
+  free_disk_space: true
```

Yeah thought there was a chance the two were related. Garbled output like that sometimes shows up when memory usage is exceeded

We could try a pagefile as well


```suggestion
  free_disk_space: true
  settings_linux:
    swapfile_size: 10GiB
```

---

Interesting, let's give it a shot. We ran out of tricks anyway...